instrumentation/log4j/log4j-appender-2.17/javaagent/build.gradle.kts

```diff
@@ -14,7 +14,7 @@ muzzle {
 val testLatestDeps = findProperty("testLatestDeps") as Boolean
 
 dependencies {
-  library("org.apache.logging.log4j:log4j-core:2.17.0")
+  library("org.apache.logging.log4j:log4j-core:2.17.2")
```

Comment:
we usually compile and test agains the earliest supported version (`library` translates to `compileOnly` + `testImplementation`)

Reply:
@laurit 
Thank you for explaining this. This makes it sounds like this PR is misguided; or am I missing something?

Reply:
> This makes it sounds like this PR is misguided

correct